jgrapht-core/src/main/java/org/jgrapht/alg/BergeGraphChecker.java

```diff
@@ -0,0 +1,1045 @@
+/*
+ * (C) Copyright 2016-2018, by Philipp S. Kaesgen and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package org.jgrapht.alg;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.jgrapht.Graph;
+import org.jgrapht.GraphPath;
+import org.jgrapht.alg.connectivity.ConnectivityInspector;
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.generate.ComplementGraphGenerator;
+import org.jgrapht.graph.AsSubgraph;
+import org.jgrapht.graph.Multigraph;
+import org.jgrapht.graph.SimpleGraph;
+
+
+/**
+ * <p>Checker for Bergeness as described in the famous paper:
+ * M. Chudnovsky, G. Cornuejols, X. Liu, P. Seymour, and K. Vuskovic. Recognizing Berge Graphs. Combinatorica 25(2): 143--186, 2003.
+ * 
+ * <p>Special Thanks to Maria Chudnovsky for her kind help.
+ * 
+ * @author Philipp S. Kaesgen (pkaesgen@freenet.de)
+ * @since 2018
+ *
+ * @param <V> the graph vertex type
+ * @param <E> the graph edge type
+ */
+public class BergeGraphChecker<V,E>{
```

Comment:
Consistent with the naming of some similar classes, I would rename this to `BergeGraphInspector`

Reply:
done.